frame/timestamp/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "GPL-3.0"
 serde = { version = "1.0.101", optional = true }
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
 sp-std = { version = "2.0.0", default-features = false, path = "../../primitives/std" }
+sp-io = { version = "2.0.0", default-features = false, path = "../../primitives/io" }
```

Comment:
Would be good if you and @shawntabrizi make imports that are needed for benchmakring be feature-gated sooner rather than later.

Reply:
That is being tracked here: https://github.com/paritytech/substrate/issues/4873